.codecov.yml

```diff
@@ -0,0 +1,16 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        target: 0%
```

Comment:
Maybe it is good to have a target?

Reply:
@mkholjuraev as far as I know this won't block PR's, and the goal was that :) 
we can control the % ourselves without any blocking codecov/patch things